frontend/pages/topics/index.jsx

```diff
@@ -0,0 +1,49 @@
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import axios from "axios";
+import Layout from "../../components/Layout";
+
+
+export default function Topics({ topicsData }) {
+  const topics = topicsData["results"] || [];
+
+  //Style for the anchor used in topicNames
+  const topicAnchorStyle = {
+    color: 'black',
+  }
+  const topicNames = topics.map((topic) => (
+    <Grid item key={topic['key']} xs={6} sm={6} md={6}>
```

Comment:
Grid breakpoints apply to all larger sizes as well, so setting `xs={6}` also sets `sm`, `md`, and `lg` to `6`, unless those values are set explicitly. If instead these are set to `xs={6}`, `sm={4}`, and `md={3}`, then in larger viewports the grid will have more columns (the grid is always divided in 12 units).

Reply:
xs has been set to 4 since I believe that 3 columns will look better. The other settings have been removed.